### PR TITLE
georgepisaltu: retain at rank 1

### PR DESCRIPTION
**Pull request by georgepisaltu**
[Rendered version](https://github.com/georgepisaltu/Evaluations/blob/main/evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md)

### CODE REVIEW

File: evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Evidence-0001: Retention at to Rank 1
```
Comment (anaelleltd) [inline evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md:1]: typo: "at to"

File: evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+| **Report Date** | Date of submission 2024/11/14                                                               |
+| **Submitted by**| George Pisaltu                                                                              |
+
+
+## Member details
+
+- Matrix username: `@george.pisaltu:parity.io`
+- Polkadot address: `128pmEUBSGjGeXZXNaAmomAJgVn77L74YT7Zdjd3fP63HWNP`
+- Current rank: `1`
+- Date of initial induction: `2024/09/09`
```
Comment (anaelleltd) [inline evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md:14]: It should read: 2024/08/16 as per [this](https://collectives-polkadot.subscan.io/extrinsic/4441984-2).

File: evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Reporting period
+
+- Start date: 2024/08/16
+- End date: 2024/11/14
+
+
+## Evidence
+
+During my first three months as a fellow, I have mainly been focused on delivering the new
```
Comment (anaelleltd) [inline evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md:27]: Edit: "as a member"

File: evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md (hunk 1 above)
Comment (georgepisaltu) [reply evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md]: Fixed.

File: evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md (hunk 2 above)
Comment (georgepisaltu) [reply evidence/georgepisaltu/0001-georgepisaltu-2024-Q3-rank1-retention.md]: Fixed.

**Review by alindima — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Review by sandreim — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]

**Review by seadanda — APPROVED** [review]

**Pull request merged**